src/main/java/io/blamer/bot/conversation/routes/Registry.java

```diff
@@ -56,67 +61,31 @@ public class Registry implements Conversation {
   @Value("${answers.registry.description}")
   private String description;
 
-  /*
-   * @todo #19 After hub service will be created, redirect the token.
-   *   Send auth message to hub, to process the data.
-   * */
+  /**
+   * The requester.
+   */
+  private final RSocketRequester requester;
```

Comment:
@l3r8yJ I suggest to name this var as `rs`
WDYT?

Reply:
@h1alexbel 2 chars will be violation for checkstyle

Reply:
@l3r8yJ sure, you are right